src/components/common/PeopleAndRoles.vue

```diff
@@ -193,10 +193,12 @@ export default class PeopleAndRoles extends Mixins(PeopleRolesMixin) {
     // assign party type (org or person)
     this.currentOrgPerson.officer.partyType = partyType
 
-    // pre-populate Completing Party's name and mailing address
+    // pre-populate Completing Party's name, email address and and mailing address
```

Comment:
This sets the CP email before the edit operation, which is different from before. Works fine.